src/github.com/getlantern/peerscanner/host.go

```diff
@@ -57,15 +59,20 @@ type status struct {
 type host struct {
 	name        string
 	ip          string
-	record      *cloudflare.Record
+	port        string
+	cflRecord   *cloudflare.Record
```

Comment:
Can you update the comment for this struct while we're at it to include CloudFront?